examples/htool.c

```diff
@@ -287,6 +287,107 @@ static int command_console(const struct htool_invocation* inv) {
   }
 }
 
+struct libhoth_device* htool_libhoth_device(void) {
+  static struct libhoth_device* result;
+  if (result) {
+    return result;
+  }
+
+  int rv;
+  const char *transport_method_str;
+  rv = htool_get_param_string(htool_global_flags(), "transport", &transport_method_str);
+  if(rv) {
+    return NULL;
+  }
+  
+  if(strlen(transport_method_str) <= 0 || 
+    (strncmp(transport_method_str, "usb", 3) == 0)) {
```

Comment:
strcmp(transport_method_str, "usb") is cleaner

Reply:
ack. Fixed